components/style/properties/shorthand/mask.mako.rs

```diff
@@ -120,6 +120,30 @@
          })
     }
 
+    fn mask_position_to_css<W>(x: &HorizontalPosition,
+                               y: &VerticalPosition,
+                               dest: &mut W)
+                               -> fmt::Result where W: fmt::Write {
```

Comment:
Do we really need a function in the first place? Can't we import `Keyword`, address the struct fields and serialize them directly? (like what `Position` does inherently)?